tests/unit/mongo.spec.ts

```diff
@@ -250,4 +251,89 @@ describe('Pipeline to mongo translator', () => {
       },
     ]);
   });
+
+  it('can generate a basic sort step on one column', () => {
+    const pipeline: Array<PipelineStep> = [
+      {
+        name: 'sort',
+        columns: ['foo'],
+        order: ['desc'],
+      },
+    ];
+    const querySteps = mongo36translator.translate(pipeline);
+    Mongo36Translator;
```

Comment:
I think this line is an error and should be removed (along with the corresponding `import` statement)